pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1651,6 +1651,10 @@ func handleModify(ctx *domainContext, key string,
 			status.ClearError()
 			publishDomainStatus(ctx, status)
 			doInactivate(ctx, status, false)
+			if status.HasError() {
+				log.Errorf("doInactivate failed: Aborting the modify of %s", status.DisplayName)
```

Comment:
Wouldn't it make sense to log the error as well here and below?

Reply:
Makes sense. Will add it.